app/controllers/people_controller.rb

```diff
@@ -0,0 +1,17 @@
+class PeopleController < ApplicationController
+
+	def index
+		@people = Person.all
+		@task = Task.all
+		@count = Task.group(:person_id).where(completed_at: nil).count
+	end
+
+	def show		
+		@id = Person.find_by(name: params[:name])
+		@people = Task.where(person_id: @id.id)
+		@completed = @people.where.not(completed_at: nil)
+		@not_completed = @people.where(completed_at: nil)
+	end
```

Comment:
I would consider refactoring these variable names. It would make more sense to have `@people` refer to a `Person` object and not a `Task`! Then you also wouldn't have to call `@id.id`.